aws/app/lambda/reliability/reliability.js

```diff
@@ -1,65 +1,41 @@
 const { getTemplateFormConfig } = require("templates");
-const { getSubmission, formatError } = require("dataLayer");
-const { LambdaClient, InvokeCommand } = require("@aws-sdk/client-lambda");
-
-const REGION = process.env.REGION;
+const { getSubmission } = require("dataLayer");
 
 exports.handler = async function (event) {
-  let submissionIDPlaceholder = "";
-
   const message = JSON.parse(event.Records[0].body);
+
   try {
     const messageData = await getSubmission(message);
-    const processedMessageData = {
-      submissionID: messageData.Item?.SubmissionID.S ?? message.submissionID,
-      formID: messageData.Item?.FormID.S ?? null,
-      sendReceipt: messageData.Item?.SendReceipt.S ?? null,
-      language: messageData.Item?.FormSubmissionLanguage.S ?? "en",
-      createdAt: messageData.Item?.CreatedAt.N ?? null,
-      formSubmission: messageData.Item?.FormData.S
-        ? JSON.parse(messageData.Item?.FormData.S)
-        : null,
-      securityAttribute: messageData.Item?.SecurityAttribute.S ?? "Unclassified",
-    };
+    const submissionID = messageData.Item?.SubmissionID.S ?? message.submissionID;
+    const formID = messageData.Item?.FormID.S ?? null;
+    const formSubmission = messageData.Item?.FormData.S ? JSON.parse(messageData.Item?.FormData.S) : null;
+    const language = messageData.Item?.FormSubmissionLanguage.S ?? "en";
+    const securityAttribute = messageData.Item?.SecurityAttribute.S ?? "Unclassified";
+    const createdAt = messageData.Item?.CreatedAt.N ?? null;
+    const sendReceipt = messageData.Item?.SendReceipt.S ?? null;
 
-    const {
-      submissionID,
-      formSubmission,
-      formID,
-      sendReceipt,
-      createdAt,
-      language,
-      securityAttribute,
-    } = processedMessageData;
-    submissionIDPlaceholder = submissionID;
     // Check if form data exists or was already processed.
     if (formSubmission === null || typeof formSubmission === "undefined") {
       // Ack and remove message from queue if it doesn't exist in the DB
       // Do not throw an error so it does not retry again
-
-      console.warn(
-        `{"status": "warn", "submissionID": "${submissionID}", "warning": "Submission data does not exist in the DB" }`
-      );
+      console.warn(JSON.stringify({
+        status: "success",
+        submissionId: submissionID,
+        sendReceipt: sendReceipt,
+        message: "Submission will not be processed because it could not be found in the database.",
+      }));
+      return { status: true };
     }
 
     // Add form config back to submission to be processed
-    formSubmission.form = await getTemplateFormConfig(formSubmission.formID);
+    formSubmission.form = await getTemplateFormConfig(formID);
+
     if (formSubmission.form === null) {
-      throw new Error(`No Form with ID ${formSubmission.formID} found to process response`);
+      throw new Error(`No associated form template (ID: ${formID}) exist in the database.`);
     }
 
-    /*
```

Comment:
Curious as to why this comment was removed?

Reply:
Oops you are right that one is still relevant :) Will put it back in.